packages/edit-post/src/index.js

```diff
@@ -177,6 +177,10 @@ export function initializeEditor(
 		} );
 	}
 
+	// Prevent the default browser action for files dropped outside of dropzones.
```

Comment:
this code, and the same in the edit site file below in my testing is only ever executed once on editor load, so I don't think we need to try and tidy up these event listeners.